src/Tables/TableCell.tsx

```diff
@@ -38,6 +40,19 @@ const StyledCell = styled.td<TableCellProps>`
   border-right: ${fromTheme('border', 'light')};
   font-family: ${fromTheme('font', 'data', 'family')};
   font-size:  ${fromTheme('font', 'data', 'size')};
+  font-weight: ${({ theme, variant }) => (
+    theme.font[(variant === TEXTVARIANT.NEGATIVE
+      || variant === TEXTVARIANT.MEDIUM)
+      ? 'bold' : 'base'].weight
```

Comment:
I guess we hadn't been setting it before, but ideally we should be using the `family`, `size`, and `weight` values from the same `FontSpec` together, so since we're using `font.data.size` and `font.data.family`, we should also be using `font.data.weight` as our default here. Currently, that value is also set to `WEIGHT.BOLD`, but I think we could update that to be `WEIGHT.MEDIUM` so it stays consistent.

(Even though this won't actually change anything in the end, I'd like to make sure our theme is as accurate as possible to save us some headaches in the long run.)

Reply:
Ok that makes sense - I updated `font.data.weight` to `WEIGHT.MEDIUM` and updated `TableCell` to use `font.data.weight` through c15f62c.